api.proto

```diff
@@ -17,6 +17,17 @@ message CreateStreamResponse {
     // Intentionally empty.
 }
 
+// DeleteStreamRequest is sent to delete a stream.
+message DeleteStreamRequest {
+    string subject           = 1; // Stream NATS subject
+    string name              = 2; // Stream name (unique per subject)
```

Comment:
Stream names are actually globally unique now, so the subject should not be needed.

Reply:
Should be fixed in 363c1bc.